src/acl_kernel_if.cpp

```diff
@@ -1138,6 +1139,7 @@ void acl_kernel_if_launch_kernel_on_custom_sof(
   // in the kern->csr_version field.  The value is cached after PLL init to
   // avoid reading back on every kernel launch, which would add overhead.
   if (kern->csr_version != CSR_VERSION_ID &&
+      kern->csr_version != CSR_VERSION_ID_20222 &&
       kern->csr_version != CSR_VERSION_ID_181) {
```

Comment:
How about using a range for future compatibility?
```suggestion
  if (!(kern->csr_version >= CSR_VERSION_ID_181 &&
        kern->csr_version <= CSR_VERSION_ID)) {
```